test/functional/JLM_Tests/src/org/openj9/test/java/lang/management/TestOpenJ9DiagnosticsMXBean.java

```diff
@@ -65,7 +65,7 @@ public class TestOpenJ9DiagnosticsMXBean {
 	private ObjectName mxbeanName = null;
 	private OpenJ9DiagnosticsMXBean diagBean = null;
 	private OpenJ9DiagnosticsMXBean diagBeanRemote = null;
-	List<String> initialDumpOptions = new ArrayList<String>();
+	String initialDumpOptions = null;
 	private JMXConnector connector = null;
```

Comment:
`initialDumpOptions` should be `private` like nearby fields; all these initializers are redundant.